# For timestamps
Header header

# Debug
bool isEnabled
float64 hopping_state
